File: Exemplos/App_IntentDigitalHUB_Flutter/Elgin_IntentDigitalHUB_Flutter-font/lib/IntentDigitalHubService/BALANCA/configurar_protocolo_comunicacao.dart
import 'package:intent_digital_hub/IntentDigitalHubService/BALANCA/balanca_command.dart';

class ConfigurarProtocoloComunicacao extends BalancaCommand {
  final int _protocoloComunicacao;

  ConfigurarProtocoloComunicacao(this._protocoloComunicacao)
      : super('ConfigurarProtocoloComunicacao');

  @override
  String functionParameters() {
    return '"protocoloComunicacao"' ':' '$_protocoloComunicacao';
  }
}

File: Exemplos M10/App_IntentDigitalHUB_Kotlin/Elgin_IntentDigitalHUB_Kotlin-font/app/src/main/java/com/elgin/intent_digital_hub/IntentDigitalHubService/BALANCA/Commands/ConfigurarProtocoloComunicacao.kt
package com.elgin.intent_digital_hub.IntentDigitalHubService.BALANCA.Commands

import com.elgin.intent_digital_hub.IntentDigitalHubService.BALANCA.BalancaCommand

class ConfigurarProtocoloComunicacao(val protocoloComunicacao: Int) :
    BalancaCommand<Int>("ConfigurarProtocoloComunicacao") {
    override fun functionParameters(): String {
        return "\"protocoloComunicacao\"" + ":" + protocoloComunicacao
    }
}